src/content/docs/logs/get-started/live-archives.mdx

```diff
@@ -96,7 +75,6 @@ A partition is a way to group logs based on specific criteria. Creating partitio
 2. To the left of the logs query bar, click <DoNotTranslate>**Data partitions**</DoNotTranslate>, then click <DoNotTranslate>**Create data partition**</DoNotTranslate>.
 3. Define a Partition name as an alphanumeric string that begins with Log_.
 4. Select the field <DoNotTranslate>**Use live archives on this partition**</DoNotTranslate>.
-5. The <DoNotTranslate>**Total effective retention**</DoNotTranslate> days will be updated to reflect the regular retention _and_ the [live archives retention](/docs/logs/get-started/live-archives-billing/#what-counts).
 
 <img
```

Comment:
@homelessbirds I'd indent this image four spaces under step 4.

Note that step 5 is not showing up as step 5, maybe because of the current indentation tells the parser to start a new list after the image.